src/makeInfinityAdder.js

```diff
@@ -5,6 +5,23 @@
  */
 function makeInfinityAdder() {
   // write code here
+  let sum = 0;
+
+  function adder(...args) {
+    if (args.length === 0) {
+      const result = sum;
+
+      sum = 0;
+
+      return result;
+    } else {
+      sum += args.reduce((acc, num) => acc + num, 0);
```

Comment:
Since the `adder` function should only accept a single argument, using `reduce` here is not necessary. You can directly add the single argument to the `sum` if it's provided.